src/general/Mutations.cs

```diff
@@ -391,6 +333,41 @@ private OrganelleDefinition GetRandomOrganelle(bool isBacteria)
             "for a new organelle");
     }
 
+    private MembraneType RandomMembraneType()
+    {
+        var simulation = SimulationParameters.Instance;
```

Comment:
This is already looked up in the function calling this, should be passed in as a parameter.